src/Impostor.Api/CheatContext.cs

```diff
@@ -0,0 +1,20 @@
+using Impostor.Server.Net.Inner;
+
+namespace Impostor.Api
+{
+    public class CheatContext
```

Comment:
What is this? Put some comments.

Reply:
Cheat context for not yet added anticheat api, basically unused for now

Reply:
Would prefer some comments, they really have been lacking (also my bad)